### DISCUSSION TITLE
Key default values not working/implemented in Classificationstore

### DISCUSSION THREAD

**Discussion by Daniel-Ateles** [pimcore/pimcore#8360] [Q&A / Support] [answered]
Hi, 

I would like to get the default values in the classificationstore keys to work, like showing them in the admin editor when adding values and saving them to db.

**EDIT 2021-03-09** Default values seems to work with most input types except the default "Input".
![Skärmklipp](https://user-images.githubusercontent.com/58037739/110428044-fea4b400-80a8-11eb-8bcb-a93eb89dde3f.PNG)

// Daniel Larsson, Ateles

**Comment by johanlindgren** [comment]
This one is really important for Ateles

**Comment by weisswurstkanone** [comment, marked as answer]
see https://github.com/pimcore/pimcore/pull/8366

**Reply by Daniel-Ateles** [reply]
Thanks! :)
